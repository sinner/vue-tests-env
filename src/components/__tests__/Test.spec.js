import { mount } from '@vue/test-utils';
import Test from '@/components/Test.vue';

describe('Test.vue', () => {
  it('renders without any prop passed', () => {
    const wrapper = mount(Test);
    expect(wrapper.html()).toMatchSnapshot();
  });
  it('renders props.value when passed', () => {
    const wrapper = mount(Test, {
      propsData: {
        value: 'Joe',
      },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
