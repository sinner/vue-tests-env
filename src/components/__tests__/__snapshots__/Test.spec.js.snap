// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test.vue renders props.value when passed 1`] = `
<div>
  <h1>Hello Joe!</h1>
</div>
`;

exports[`Test.vue renders without any prop passed 1`] = `
<div>
  <h1>Hello Test!</h1>
</div>
`;
